main.tf

```diff
@@ -122,30 +113,20 @@ resource "azurerm_key_vault_access_policy" "vault_init" {
 }
 
 module "vault_identity" {
-  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/identity"
+  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/aad-pod-identity/identity"
 
-    kubectl_host                   = var.kubectl_host
-    kubectl_client_certificate     = var.kubectl_client_certificate
-    kubectl_client_key             = var.kubectl_client_key
-    kubectl_cluster_ca_certificate = var.kubectl_cluster_ca_certificate
-
-    identity_name        = azurerm_user_assigned_identity.vault.name
-    identity_client_id   = azurerm_user_assigned_identity.vault.client_id
-    identity_resource_id = azurerm_user_assigned_identity.vault.id
+  identity_name        = azurerm_user_assigned_identity.vault.name
+  identity_client_id   = azurerm_user_assigned_identity.vault.client_id
+  identity_resource_id = azurerm_user_assigned_identity.vault.id
 
 }
 
 module "vault_init_identity" {
-  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/identity"
-
-    kubectl_host                   = var.kubectl_host
-    kubectl_client_certificate     = var.kubectl_client_certificate
-    kubectl_client_key             = var.kubectl_client_key
-    kubectl_cluster_ca_certificate = var.kubectl_cluster_ca_certificate
+  source = "/Users/tmiller/gitlab-repos/tfe/terraform-azurerm-kubernetes/aad-pod-identity/identity"
```

Comment:
Should this be ${path.module}

Reply:
yep, was a remnant from development, thanks